src/devices/Pn532/samples/Program.cs

```diff
@@ -164,19 +164,21 @@ void ReadMiFare(Pn532 pn532)
         {
             mifareCard.BlockNumber = block;
             mifareCard.Command = MifareCardCommand.AuthenticationB;
-            var ret = mifareCard.RunMifiCardCommand();
+            var ret = mifareCard.RunMifareCardCommand();
+            // This will reselect the card in case of issue
+            mifareCard.ReselectCard();
```

Comment:
should this be automatically called from RunMifareCardCommand?

Reply:
It depends what you want to do, so I give multiple ways of running a command